src/types.js

```diff
@@ -6,21 +6,38 @@ export type QuiqChatSettings = {
   BURNED?: boolean,
 };
 
-export type EventType = 'Text' | 'Join' | 'Leave';
+export type EventType = 'Join' | 'Leave' | 'Register';
 export type AuthorType = 'Customer' | 'Agent';
-export type MessageType = 'Text' | 'ChatMessage' | 'BurnItDown';
+export type TextMessageType = 'Text';
```

Comment:
Ok. I kind of did this for uniformity to match with the MessageTypes that do have more than a single enumeration. Also, in case there's eventually more than one type of `TextMessage`. But let me know if I'm overthinking it ;)